jenkins/Jenkinsfile

```diff
@@ -119,6 +119,37 @@ pipeline {
                         sh 'docker logout quay.io'
                     }
                 }
+                stage('Upload-PR-to-SonarQube') {
+                    agent { label 'docker' }
+                    environment {
+                        SONAR_HOST_URL = credentials('CANDLEPIN_SONAR_HOST_URL')
```

Comment:
I added this `SONAR_HOST_URL` as a global env var, you can use it directly and we don't have to do this credentials lookup.